<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filling</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonInFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>1081</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>1081</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFill">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>1121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditProbeLen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHitLimit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Probe length (um)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Inside hit limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of iterations</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNiter">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>471</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVoxelsize_x">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVoxelsize_z">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_voxel_x">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Voxel size x (um)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_voxel_z">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Voxel size z (um)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVoxelsize_y">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_voxel_y">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Voxel size y (um)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonInFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonFill</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>filler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inFileSelecter()</slot>
  <slot>outFileSelecter()</slot>
  <slot>filler()</slot>
 </slots>
</ui>
